src/components/titlePane/styles/titlePane.css

```diff
@@ -0,0 +1,26 @@
+@import '../../../styles/variables.css';
+
+.main {
+	overflow: hidden;
+}
+
+.title {
+	background-color: var(--title-bar-bg);
+	border-color: var(--title-bar-bg);
+	border-style: solid;
+	border-width: 1px;
```

Comment:
This should be a CSS variable, e.g. `var(--title-pane-border-width)`. Don't worry about getting specific.